.github/workflows/format.yml

```diff
@@ -0,0 +1,28 @@
+name: dotnet-format
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+on:
+  pull_request:
+    branches:
+    - '**'      # matches every branch
```

Comment:
This can be simplified to `on: [pull_request]` unless there another reason we need to filter by branches which can still be a good feature.